src/components/Menu/index.js

```diff
@@ -30,6 +30,20 @@ function Menu() {
         <li>
           <Link to="/visits/edit/test-id">Edit a visit</Link>
         </li>
+        <li>
+          <Link to="/patients">Patients - Will display a table of all patients</Link>
+        </li>
+        <li>
+          <Link to="/patientdetails">Patient Details - Will display the details of a patient.</Link>
```

Comment:
The link is wrong here. Follow the visits pattern and change this link to point to the correct page